src/rprblender/engine/image_filter.py

```diff
@@ -24,6 +24,35 @@
 from rprblender.utils.user_settings import get_user_settings
 
 
+# used in Reflection Catcher calculations
+FILTER_CODE_COLOR_WITH_REFLECTION = """
+int2 coord;
+GET_COORD_OR_RETURN(coord, GET_BUFFER_SIZE(outputImage));
+vec4 pixel = ReadPixelTyped(inputImage, coord.x, coord.y);
+vec4 pixel_alpha = ReadPixelTyped(alphaImage, coord.x, coord.y);
+vec4 pixel_background = ReadPixelTyped(backgroundImage, coord.x, coord.y);
+vec4 pixel_alpha_neg = pixel_alpha;
+pixel_alpha_neg.x = 1.0f - pixel_alpha.x;
+
+pixel.xyz = pixel.xyz * pixel_alpha.x + pixel_background.xyz * pixel_alpha_neg.x;
+WritePixelTyped(outputImage, coord.x, coord.y, pixel);
+"""
+
+# used in Shadow catcher calculations
+FILTER_CODE_COLOR_WITH_SHADOW = """
+int2 coord;
+GET_COORD_OR_RETURN(coord, GET_BUFFER_SIZE(outputImage));
+vec4 pixel = ReadPixelTyped(inputImage, coord.x, coord.y);
+vec4 pixel_alpha = ReadPixelTyped(alphaImage, coord.x, coord.y);
+__constant vec4 shadow_color = (vec4)(0.0f, 0.0f, 0.0f, 1.0f);
+vec4 pixel_alpha_neg = pixel_alpha;
+pixel_alpha_neg.x = 1.0f - pixel_alpha.x;
+
+pixel.xyz = pixel.xyz * pixel_alpha.x + shadow_color.xyz * pixel_alpha_neg.x;
+WritePixelTyped(outputImage, coord.x, coord.y, pixel);
+"""
```

Comment:
Move this constants to ImageFilterTransparentShadowReflectionCatcher.

Reply:
@ardraeiss This change was requested 8 days ago.  Have you addressed?